### PR TITLE
fix(docs): Update HOWTO-hi.md with current CI system

### PR DESCRIPTION
**Pull request by akshat-rawat**
## What does this PR do?
Improve repo

### Description
Updating localization with GitHub Actions as described in #5644

### Why is this valuable (or not)?
Helps go ahead with #5644

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/master/CONTRIBUTING.md)
- [x] Search for duplicates.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: HOWTO-hi.md [hunk 1]
```diff
@@ -9,6 +9,6 @@
 
 सवाल पूछने में संकोच न करें; हर योगदानकर्ता ने पहले PR के साथ शुरुआत की। आप हमारे हजारवें हो सकते हैं!
 
-यहां तक ​​कि अगर आप एक अनुभवी ओपन सोर्स योगदानकर्ता हैं, तो ऐसी चीजें हैं जो आपको यात्रा कर सकती हैं। एक बार जब आप अपना पीआर सबमिट कर देते हैं, तो ट्रैविस-सीआई एक लाइनर चलाएगा, अक्सर रिक्ति या वर्णमाला के साथ छोटे मुद्दों को ढूंढता है। यदि आपको एक हरा बटन मिलता है, तो सब कुछ समीक्षा के लिए तैयार है, लेकिन यदि नहीं, तो यह जानने के लिए ट्रैविस लिंक पर क्लिक करें कि लिंटर को क्या पसंद नहीं था। समस्या को ठीक करें और अपने पीआर के लिए एक प्रतिबद्धता जोड़ें।
+यहां तक कि अगर आप एक अनुभवी ओपन सोर्स योगदानकर्ता हैं, तो ऐसी चीजें हैं जो आपको यात्रा कर सकती हैं। एक बार जब आप अपना पीआर सबमिट कर देते हैं, तो गिथब एक्शन एक लिंटर चलाएगा, अक्सर रिक्ति या वर्णमाला के साथ छोटे मुद्दों को ढूंढता है। यदि आपको एक हरा बटन मिलता है, तो सब कुछ समीक्षा के लिए तैयार है, लेकिन यदि नहीं, तो यह जानने के लिए फेल्ड चेक के नीचे "डिटेल्स" पर क्लिक करें कि लिंटर को क्या पसंद नहीं आया। समस्या को ठीक करें और अपने पीआर के लिए एक प्रतिबद्धता जोड़ें।
```
Comment (davorpa) [inline HOWTO-hi.md:12]: Could you give without translation the "GitHub Actions" term? It's a requirement of the issue

> Most instances of the change were simple as we only replaced the noun; therefore nothing should've needed to change. (This is with the assumption that "GitHub Actions" should be treated as a noun and is not localized.)

**Review by davorpa — CHANGES_REQUESTED** [review]

File: HOWTO-hi.md (hunk 1 above)
Comment (eshellman) [reply HOWTO-hi.md:12]: or explain the change?

Comment (SethFalco) [reply HOWTO-hi.md:12]: Actually, when I said that I didn't intend for it to be a firm rule. ^-^'
I was just saying what we did, which was with the assumption that nouns don't change.

I believe it's common for languages that have a different character set to modify formal nouns (like "Seth" or "GitHub", not "table"). I'm just using Urdu as an example, but I know with that, nouns are translated by pronunciation to the other character set.

Just verified with a friend who knows Bosnian (can be written in Latin and Cyrillic) and in bs_Cyrl translations, nouns are _often_ translated to use Cyrillic characters.

Comment (davorpa) [reply HOWTO-hi.md:12]: @SethFalco then go ahead?

Comment (akshat-rawat) [reply HOWTO-hi.md:12]: Should I change गिथब एक्शन to "गिथब एक्शन" or to "GitHub Actions"?

**Pull request merged**